packages/mui-codemod/src/deprecations/backdrop-props/test-cases/expected.js

```diff
@@ -11,17 +11,16 @@ import { Backdrop as MyBackdrop } from '@mui/material';
   slots={{
     root: 'div',
     transition: CustomTransition
-  }}
-  slotProps={{
-    root: { className: 'foo' },
```

Comment:
`slotProps` is not modified in the `backdrop-props` codemod, so there's no need for it on the test case. Same for line 22.